fdbclient/FileBackupAgent.actor.cpp

```diff
@@ -1066,8 +1551,8 @@ struct BackupRangeTaskFunc : BackupTaskFuncBase {
 		return _finish(tr, tb, fb, task);
 	};
 
-	// Finish (which flushes/syncs) the file, and then in a single transaction, make some range backup progress durable.
-	// This means:
+	// Finish (which flushes/syncs) the file, and then in a single transaction, make some range backup progress
+	// durable. This means:
```

Comment:
nit: what is the proposed change here?

Reply:
I think this was a pretty old file so devFormat fixed some formatting issues